aws/src/test/java/org/apache/iceberg/aws/s3/TestS3InputStream.java

```diff
@@ -52,18 +52,15 @@ public void before() {
     createBucket("bucket");
   }
 
-  @Test
```

Comment:
Mistake here in my recent push this morning, the test method needs to be called passing in the client.